portfolio/src/main/webapp/index.html

```diff
@@ -27,15 +30,18 @@ <h2>I'm currently a STEP intern at Google.</h2>
     </div>
     
     <div class="about">
-      <h1>About</h1> 
-      <p>I'm a rising junior at the University of Pennsylvania studying computer science. 
-         I'm currently working as a STEP intern for Google. This year the internship 
-         consists of technical training projects and a capstone project. Check out my 
-         Github to see what I'm currently working on! 
+      <h1 id="about-text">About</h1> 
```

Comment:
The `id` attribute should be unique on the page, `class` may be a better fit here.